lib/uwb/include/uwb/UwbDevice.hxx

```diff
@@ -55,6 +55,14 @@ public:
     ::uwb::protocol::fira::UwbDeviceInformation
     GetDeviceInformation();
 
+    /**
+     * @brief Get the number of sessions associated with the device.
+     *
+     * @return std::optional<uint32_t>
+     */
+    std::optional<uint32_t>
```

Comment:
I have no idea what compelled us to use `std::optional<uint32_t>` here since `!has_value()` is essentially equivalent to 0, so now there is ambiguity as to what each of those things mean. I also noticed both `I*Ddi` interfaces have entries for `IOCTL_UWB_GET_SESSION_COUNT`. So, I'll fix these in a follow-up PR.

Reply:
#213